feature_layers/feature-collection-layer-from-portal/src/main/java/com/esri/samples/feature_collection_layer_from_portal/FeatureCollectionLayerFromPortalSample.java

```diff
@@ -58,45 +58,38 @@ public void start(Stage stage) {
             stage.setScene(scene);
             stage.show();
 
-
-            // create a new ArcGISMap with an oceans Basemap
+            // create a new ArcGISMap with the oceans basemap
             map = new ArcGISMap(Basemap.createOceans());
-            
-            // create a map view and set the ArcGISMap to it
+
+            // create portal
+            portal = new Portal("https://www.arcgis.com/");
+
+            // create a map view and set the map to it
             mapView = new MapView();
             mapView.setMap(map);
 
             // create text field to input user's own portal item ID
-            input = new TextField();
-            input.setMaxWidth(250);
-            input.setText("32798dfad17942858d5eef82ee802f0b");
-
-            // create button to perform action
-            Button fetchFromPortal = new Button("Open from portal item");
-            fetchFromPortal.setMaxWidth(250);
-
-             // verify the input and fetch the portal item
-            fetchFromPortal.setOnAction(e -> {
-
-                if (!"".equals(input.getText())) {
-                    fetchFeatureCollectionFromPortal();
-                } else {
-                    new Alert(Alert.AlertType.ERROR, "Portal Item ID is empty. Please enter a Portal Item ID.").show();
-                }
-
-            });
+            inputTextField = new TextField();
+            inputTextField.setMaxWidth(250);
+            inputTextField.setText("32798dfad17942858d5eef82ee802f0b");
 
             fetchFeatureCollectionFromPortal();
```

Comment:
Let's add a small comment here to help the users see what's going on:

```suggestion
			// fetch the default feature collection layer from the portal
            fetchFeatureCollectionFromPortal();
```